scripts/devel/generate_devel_script.py

```diff
@@ -69,16 +73,24 @@ def beforeInclude(self, *args, **kwargs):
                         'fi',
                     ]
                     script = '\n'.join(lines)
+                if args.build_tool and ' --build-tool ' in script:
+                    script = script.replace(
```

Comment:
This seems fragile for longer term maintenance.

Reply:
It is, but I don't see a good way to change this at the moment. Proposals how to refactor it are of course welcome.